README.md

```diff
@@ -12,10 +12,51 @@ clone this repo, and make sure you have build-essentials installed. Go into the
 
 Then you can find `hyper-initrd.img` in the build directory, together with a pre-built kernel.
 
+### Build for VirtualBox
 If you want to get the boot disk file for VirtualBox, please reconfigure with flag --with-vbox,
 
     > ./configure --with-vbox
     > make
 
 Then you can find `hyper-vbox-bootimage.iso` in the build directory. Booting from this iso will
 bring you to the hyper world.
+
+### Build Cbfs rom
+If you want to get the cbfs.rom file to speed up booting. please install `cbfstool` first and
```

Comment:
We should remove all cbfs related scripts/docs, since they are not working any more.